test/functional/rpc_psbt.py

```diff
@@ -87,10 +87,11 @@ def run_test(self):
         new_psbt = self.nodes[0].converttopsbt(rawtx['hex'])
         self.nodes[0].decodepsbt(new_psbt)
 
-        # Make sure that a psbt with signatures cannot be converted
+        # Make sure that a non-psbt with signatures cannot be converted
+        # Error could be either "TX decode failed" (segwit inputs causes parsing to fail) or "Inputs must not have scriptSigs and scriptWitnesses"
```

Comment:
segwit references

Reply:
since Dash don't use segwit, do you the PR needs to be backported? The error message is valid here

Reply:
yeah, should just drop this

Reply:
PR #14818 has been dropped(not valid)